alf/networks/projection_networks.py

```diff
@@ -663,7 +665,8 @@ def __init__(self,
             self._scale = nn.Parameter(
                 action_spec.constant(scale_bias_initializer_value),
                 requires_grad=True)
-            self._scale_projection_layer = lambda _: self._scale
+            self._scale_projection_layer = lambda x: expand_batch_dim(
```

Comment:
Why not just use `tensor_extend_new_dim(self._scale, 0, x.shape[0])`?

Reply:
changed to use tensor_extend_new_dim